### DISCUSSION TITLE
OnlyT QR code not functioning at KH

### DISCUSSION THREAD

**Discussion by MichelFouchard** [AntonyCorbett/OnlyT#451] [Q&A]
We love this application and were using it before Covid, and want to thank Anthony and all who worked to make it so nice.
I lately instaled the new version at home, before we resume meetings at JH, and all was OK.
But the QR code iis not working at KH.
At home, both the computer and the phone are on the same wifi network.
I thought it was the same at KH. the only difference is that the PC is directly connected at the ADSL box, while the phone is connected to this adsl box via Wifi.
Could that might be the reason why it does not work ?
Anything I should change or try ?  
Thank you for your help,
Michel
KH located  in Haut Doubs, 1 hour away from Switzerland.  :)

**Comment by AntonyCorbett** [comment]
@MichelFouchard Hello Michel. Thanks for your feedback. A few basic questions:

1. Are you using the QR code from the KH PC, rather than relying on the link provided by your home PC?
2. Have you opened the port on the KH PC using the OnlyT **Remote Access** settings?
3. Is the **Enabled** checkbox checked in the **Web Clock** settings?
4. When you click the web link in the **Web Clock** section, does it show the clock in your PC's web browser?

Regards
Antony

**Comment by AntonyCorbett** [comment]
> I thought it was the same at KH. the only difference is that the PC is directly connected at the ADSL box, while the phone is connected to this adsl box via Wifi. Could that might be the reason why it does not work ?

No, I don't think so

**Comment by MichelFouchard** [comment]
Hi Anthony,Thank you for this tools, this is really appreciated !Sorry it took me so long to come back to you, my wife had the flu so we were not able to go to the kingdom hallThanks to your questions, I realize I probably missed to select enable the clock, iI think it was ok for the portnow, all works fine.
but your questions make me asking you something else.I see there is a mobile application.can this be used at the kingdom hall to monitor the chronometer, and remove a part of the work for the brother at the audio video equipment ?  and give it to another brother ?
Wit all our christian love,
Michel

    Le vendredi 15 avril 2022, 09:55:16 UTC+2, Antony Corbett ***@***.***> a écrit :  
 
 


@MichelFouchard Hello Michel. Thanks for your feedback. A few basic questions:
   
   - Are you using the QR code from the KH PC, rather than relying on the link provided by your home PC?
   - Have you opened the port on the KH PC using the OnlyT Remote Access settings?
   - Is the Enabled checkbox checked in the Web Clock settings?
   - When you click the web link in the Web Clock section, does it show the clock in your PC's web browser?

Regards
Antony

**Reply by AntonyCorbett** [reply]
> can this be used at the kingdom hall to monitor the chronometer, and remove a part of the work for the brother at the audio video equipment ?

Yes - exactly
